### PR TITLE
Update e2e tests to reflect recent changes

### PR DESCRIPTION
**Pull request by tejohnso**
@fjvallarino please review

### CODE REVIEW

File: test/e2e/storage-sign-in.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = function(driver, LOCALCLIENT, LOCALSERVER, USER, PASSWORD) {
   passwordLocator = {"id": "Passwd"},
   googleSignInLocator = {"id": "signIn"},
   localApproveLocator = {"id": "submit_approve_access"},
-  trashFileLocator = {"css": "a[title='Trash']"};
+  trashFileLocator = {"css": "span.folder.ng-binding"};
```
Comment (fjvallarino) [inline test/e2e/storage-sign-in.js]: There is an issue with this selector. It applies to any folder, so if we already have one folder created when this tests runs, it won't necessarily pick Trash.

Comment (tejohnso) [inline test/e2e/storage-sign-in.js]: Should be ok.  The purpose of the test is to verify that the sign-in worked and a file listing was received.  So any file will do.

Actually, the problem is that the *name* is wrong now.  I'll fix that.

**Comment by tejohnso** [comment]
@fjvallarino I've updated the variable name to reflect its purpose.  Note that the trash folder will always exist, so any time the file listing works, at least one folder will be found.

**Comment by fjvallarino** [comment]
:+1:

**Pull request merged**